### PR TITLE
feat(net.configuration): Updated metatype with new IPv6 properties

### PR DESCRIPTION
**Pull request by sfiorani**
**Brief description of the PR:**

This PR updates those methods that generate metatype for network interfaces, in order to update also the new IPv6 protocol configuraions.

**Related Issue:**

**Description of the solution adopted:**

**Screenshots:**

**Manual Tests**: 

**Any side note on the changes made:**

### CODE REVIEW

**Review by pierantoniomerlino — APPROVED** [review]

File: kura/org.eclipse.kura.net.configuration/src/main/java/org/eclipse/kura/net/configuration/NetworkConfigurationPropertyNames.java [hunk 1]
```diff
@@ -22,8 +22,15 @@ public enum NetworkConfigurationPropertyNames {
     CONFIG_IPV4_ADDRESS,
     CONFIG_IPV4_PREFIX,
     CONFIG_IPV4_GATEWAY,
-    CONFIG_DNS_SERVERS,
```
Comment (MMaiero) [inline kura/org.eclipse.kura.net.configuration/src/main/java/org/eclipse/kura/net/configuration/NetworkConfigurationPropertyNames.java:25]: Isn't this a breaking change? @pierantoniomerlino

Comment (pierantoniomerlino) [reply kura/org.eclipse.kura.net.configuration/src/main/java/org/eclipse/kura/net/configuration/NetworkConfigurationPropertyNames.java:25]: @MMaiero The package is exported as internal. Is it still a breaking change?

Comment (MMaiero) [reply kura/org.eclipse.kura.net.configuration/src/main/java/org/eclipse/kura/net/configuration/NetworkConfigurationPropertyNames.java:25]: if it has implications to the snapshot, probably it is a problem, right?

Comment (sfiorani) [reply kura/org.eclipse.kura.net.configuration/src/main/java/org/eclipse/kura/net/configuration/NetworkConfigurationPropertyNames.java:25]: @marcellorinaldo had similar doubt while development

Comment (pierantoniomerlino) [reply kura/org.eclipse.kura.net.configuration/src/main/java/org/eclipse/kura/net/configuration/NetworkConfigurationPropertyNames.java:25]: AFAIK, this enum is used only to retrieve the Messages in the `/resources /.../NetworkConfigurationMessagesBundle.properties`, so I think that we can change it. I'd to admit that the name of the enum is a bit misleading.

Comment (MMaiero) [reply kura/org.eclipse.kura.net.configuration/src/main/java/org/eclipse/kura/net/configuration/NetworkConfigurationPropertyNames.java:25]: ok for me if we have the confidence that this is not going to be an issue for users that will use old snapshots in this new version

**Review by pierantoniomerlino — APPROVED** [review]

**Pull request merged**